### ISSUE TITLE
Please elaborate on the comment given in "-Source"

### ISSUE THREAD

**Issue by nils-a** [MicrosoftDocs/office-docs-powershell#1543]
The explanation of the "-Source"-parameter contains the text 

&gt; The following are valid sources...

However, the sources given there do not correlate to the values of the "-Source"-parameter. 
What is meant by that comment?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a8c8a25e-c9c4-e959-00cc-e79915a9c962
* Version Independent ID: b12fef77-6a75-a078-886a-da48233dde18
* Content: [Import-SPAppPackage (sharepoint-server)](https://docs.microsoft.com/en-us/powershell/module/sharepoint-server/import-spapppackage?view=sharepoint-ps)
* Content Source: [sharepoint/sharepoint-ps/sharepoint-server/Import-SPAppPackage.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/sharepoint/sharepoint-ps/sharepoint-server/Import-SPAppPackage.md)
* Product: **sharepoint-2016-powershell**
* GitHub Login: @techwriter40
* Microsoft Alias: **kirks**

**Comment by tseward** [comment]
Looks like this is a duplicate; I responded to #1544.

**Comment by Techwriter40** [comment]
closing comment, as @tseward addresses this in comment:  https://github.com/MicrosoftDocs/office-docs-powershell/issues/1544

**Comment by nils-a** [comment]
Maybe I wrote this too broadly. *This* Question really was:
Can Someoone please change the docs so that the regarding the `-Source` parameter is less ambiguous.
i.e. Is it possible to clarify the following (in my opinion contradictory) parts:

First part:
> The following are valid values:
> InvalidSource, Marketplace, CorporateCatalog, DeveloperSite, ObjectModel, RemoteObjectModel

Second Part
> The following are valid sources (not parameters of -Source) of the app:
> --SharePoint Store
> --App catalog
> --SharePointService- Indicates apps that were built in place with SharePoint features, for example Access Services.

**Comment by nils-a** [comment]
Thanks, @tseward that's better.
However.... the block still states 

> The following are valid values: [...] 
> The following are valid sources: [...]

And in fact it means that while the former six values are valid (from the enum-point-of-view), only the latter three can be used, right? Can that be stated a bit less confusing?

**Comment by AndreaBarr** [comment]
@nils-a Thank you for this feedback

I am going to make sure the blocks that are still not correct get over to the PowerShell team for updating. 

I was able to get this into the backlog for the PowerShell team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.
